### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by thiskevinwang**
# Description

This updates the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,149 @@
...
+  - This command will run the website locally using a locally built image
+  - This includes live reload which will load your changes as you make them.
+- `make website/build-local`
+  - This command will build a local image of the website from `hashicorp/dev-portal.git`.
+
+...and then visit `http://localhost:3000`.
+There's no need to re-run `make website` each time the site is run, only the
+first time.
+
+<!-- BEGIN: editing-docs-sidebars -->
```
Comment (thiskevinwang) [inline README.md:28]: Only using the `editing-docs-sidebars` markdown block for now

**Review by BRKalow — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,465 @@
...
+  Its worth noting that _only the inline code at the beginning of the list item_ will cause problems if changed. So if you changed the above markup to...
+
+  ```markdown
+  - lsdhfhksdjf
+  - `this` jsdhfkdsjhkdsfjh
+  ```
+
+  ...while it perhaps would not be an improved user experience, no links would break because of it. The best approach is to **avoid changing headlines and inline code at the start of a list item**. If you must change one of these items, make sure to tag someone from the digital marketing development team on your pull request, they will help to ensure as much compatibility as possible.
+
+### Custom Components
```
Comment (laurapacilio) [inline README.md:85]: can we remove this and just have what we have in the terraform website repo? I think we only want this: 

Custom Components
A number of custom [mdx components](https://mdxjs.com/) are available for use within any .mdx file. If you have questions about custom components, or have a request for a new custom component, please reach out to @hashicorp/digital-marketing.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,465 @@
...
+      {
+        "title": "File",
+        "path": "indexless-category/file"
+      }
+    ]
+  }
+]
+```
+
+### Custom or External Links
```
Comment (laurapacilio) [inline README.md:421]: this seems like a repeat? Do we already have "Custom or External Links" above?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,465 @@
 # terraform-docs-common
 Content for Terraform's documentation.
+
+- https://terraform.io/cloud-docs
+- https://terraform.io/plugin
+- https://terraform.io/registry
+
+## Running the Site Locally
```
Comment (laurapacilio) [inline README.md:8]: Could we please also have the following (similar to what we have on terraform-website?) 
- Table of Contents
- Contributions Welcome!
- Where the Docs Live

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Reallyy close! Could we make these changes here and in the other agents repo as well?

File: README.md (hunk 2 above)
Comment (thiskevinwang) [reply README.md:146]: Updated!

File: README.md (hunk 3 above)
Comment (thiskevinwang) [reply README.md]: Thanks for catching this! Removed the duplicated section

File: README.md (hunk 4 above)
Comment (laurapacilio) [reply README.md:69]: that's fair and I support!

**Review by laurapacilio — APPROVED** [review]
thank you!!

**Pull request merged**